### PR TITLE
hpe3par document type

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Add in `type` documentation that was missed in https://github.com/ansible/ansible/pull/39656/

Also `version_added` should be a string.

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Review by thedoubl3j — APPROVED** [review]
lgtm, sorry we missed that.

**Comment by farhan7500** [comment]
Thanks!

Will ensure that these changes are taken care of in the future PRs

File: lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py [hunk 1]
```diff
@@ -65,23 +72,26 @@
...
     required: true
+    type: str
   secure:
     description:
       - Specifies whether the certificate needs to be validated while communicating.
     type: bool
     default: no
 extends_documentation_fragment: hpe3par
-version_added: 2.8
+version_added: "2.8"
```
Comment (acozine) [inline lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py]:
```suggestion
version_added: '2.8'
```

Comment (acozine) [reply lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py]: We're changing "" to '' elsewhere, might as well nip it in the bud here.

Comment (acozine) [reply lib/ansible/modules/storage/hpe3par/ss_3par_cpg.py]: Ahem. I meant, changing "string" to 'string' elsewhere . . .

**Pull request merged**